tests/shared/robot.spec.js

```diff
@@ -131,16 +131,16 @@ describe('Shared: Robot', function() {
         data: {
           message: 'foo',
           code: 'MYERROR',
-          stacktrace: error.stack
+          stack: error.stack
         }
       });
     });
 
     it('should handle a string error', function() {
```

Comment:
> I'd like to avoid ... checks all over the place

It's not "all over the place" - you'll see I pointed out above that `robot.printError` is only used in two places.

> I edited the signatures to say we only accept Errors (to kinda enforce that), but the function will still handle other values, just in case.

Fair enough. But if `robot.printError` accepts a string "just in case", then you ought to do the same for `utils.printError` because it appears just a couple of lines below `robot.printError` in `lib/cli/etcher.js` (with the same `error` variable being passed to both).